config/default.yml

```
@@ -701,6 +701,11 @@ Rails/ReversibleMigrationMethodDefinition:
   Include:
     - db/migrate/*.rb
 
+Rails/RootJoinChain:
+  Description: 'Use a single `#join` instead of chaining on `Rails.root` or `Rails.public_path`.'
+  Enabled: pending
+  VersionAdded: '2.13'
```

Can you specify `'<<next>>'` instead of?

```suggestion
  VersionAdded: '<<next>>'
```